dns_pre_lda.scala

```diff
@@ -78,51 +79,97 @@ val l_top_domains = Source.fromFile("top-1m.csv").getLines.map(line => {
                     }).toSet
 val top_domains = sc.broadcast(l_top_domains)
 
-/**
-val file_list = System.getenv("DPATH")    //This can be one file or many file pathes separated by commas
-val domain_table = System.getenv("DOMAIN_TABLE")
-val feedback_file = System.getenv("FPATH")
-val duplication_factor = System.getenv("DUPFACTOR").toInt
-val output_file = System.getenv("HPATH") + "/dns_word_counts"
-val output_file_for_lda = System.getenv("HPATH") + "/dns_lda_word_counts"
-*/
-
-//-----------------------------
+case class Feedback(frame_time: String,
+                        unix_tstamp: String,
+                        frame_len: Int,
+                        ip_dst: String,
+                        dns_qry_name: String,
+                        dns_qry_class: String,
+                        dns_qry_type: String,
+                        dns_qry_rcode: String,
+                        dns_sev: Int)
+val scoredFileExists = new java.io.File(feedback_file).exists
+
+var falsepositives : org.apache.spark.rdd.RDD[String]  = if (scoredFileExists){
+    /* dns_scores.csv - feedback file structure
+    
+    0   frame_time             object
+    1   frame_len              object
+    2   ip_dst                 object
+    3   dns_qry_name           object
+    4   dns_qry_class           int64
+    5   dns_qry_type            int64
+    6   dns_qry_rcode           int64
+    7   domain                 object
+    8   subdomain              object
+    9   subdomain_length        int64
+    10  num_periods             int64
+    11  subdomain_entropy     float64
+    12  top_domain              int64
+    13  word                   object
+    14  score                 float64
+    15  query_rep              object
+    16  hh                      int64
+    17  ip_sev                  int64
+    18  dns_sev                 int64
+    19  dns_qry_class_name     object
+    20  dns_qry_type_name      object
+    21  dns_qry_rcode_name     object
+    22  network_context       float64
+    23  unix_tstamp             
+    */    
+    val FrameTimeIndex = 0
```

Comment:
only classe and objects should be capitalized, eg. frameLenIndex and dnsQryNameIndex

also, correct camelcasing would be unixTStampIndex and honestly, why not spell it out as unixTimeStampIndex ?